# HG changeset patch
# User Pulkit Goyal <pulkit@yandex-team.ru>
# Date 1542291052 -10800
# Node ID b06be581569214a07a2f309ee3287a0c5e6c5793
# Parent  04a24a13960ff1545a49707541e748de191304dd
README: update links and mercurial version supported

The README still had old links to selenic.com. This patch changes them, and also
updates the mercurial versions it supports.

diff --git a/README b/README
--- a/README
+++ b/README
@@ -16,19 +16,17 @@
 ------------
 You need to have either have Subversion 1.5 (or later) installed along with
 either Subvertpy 0.7.4 (or later) or the Subversion SWIG Python bindings. You
-need Mercurial 2.0 or later.
+need Mercurial 3.7 or later.
 
-.. _mercurial: http://selenic.com/repo/hg
-.. _mercurial-stable: http://selenic.com/repo/hg-stable
-.. _crew: http://hg.intevation.org/mercurial/crew
-.. _crew-stable: http://hg.intevation.org/mercurial/crew-stable
+.. _mercurial: http://mercurial-scm.org/repo/hg
+.. _mercurial-stable: http://mercurial-scm.org/repo/hg-stable
 
 If you are unfamiliar with installing Mercurial extensions, please see
 the UsingExtensions_ page in the Mercurial wiki. Look at the example
 for specifying an absolute path near the bottom of the page. You want
 to give the path to the top level of your clone of this repository.
 
-.. _UsingExtensions: http://mercurial.selenic.com/wiki/UsingExtensions
+.. _UsingExtensions: http://mercurial-scm.org/wiki/UsingExtensions
 
 Before using hgsubversion, I *strongly* encourage you to run the
 automated tests. Just use nose_ if you have it (or ``easy_install
